releasenotes/notes/53402.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: security
+issue:
+  - 53402
+
+releaseNotes:
```

Comment:
Should prob be an upgrade note?

Reply:
Do we define upgrade notes in the istio repo?

Reply:
Yep; instead of the releaseNote key you see here; use the upgradeNote: key. There should be examples in the releasenotes/ directory

Reply:
Updated